### DISCUSSION TITLE
Access to previous nextflow documentation versions

### DISCUSSION THREAD

**Discussion by arnaudbore** [nextflow-io/nextflow#4499] [Q&A]
Hi,

Quick question about nextflow documentation. Is there a way to access the previous version of nextflow ?
The url says **latest** but I didn't find a way to show the old version of the documentation (<22.x.x for example).

Thank you in advance
Arnaud

**Comment by bentsherman** [comment]
Currently we only publish docs for the latest stable release and edge release. You can build older versions of the docs by cloning any of the `STABLE-*` branches and building/viewing locally. Eventually we will probably do that ourselves, it's just that the docs themselves have undergone massive changes over the past few years in overall structure, beyond the usual feature updates.

For what it's worth, we try to add a version note to new features so that you can always tell from the current docs whether something is available in an older version.

See also #3458
